### DISCUSSION TITLE
Prefill custom rights field permissions with minimal permissions

### DISCUSSION THREAD

**Discussion by paulboudewijn** [directus/directus#9882] [Feature Requests]
When creating a new role, the custom right field permissions are only prefilled with the recommended permissions, not the minimal permissions. There is just a textual warning below those fields showing the minimal permissions as JSON. 

**default custom read rights of `directus_users`**
![image](https://user-images.githubusercontent.com/17253180/142193552-ba4c7b05-2fdb-4b1d-b1e2-6baba0156bf3.png)

**default custom update rights of `directus_users`**
![image](https://user-images.githubusercontent.com/17253180/142193925-e0f32ef9-7240-4375-8c77-0e7532cc58af.png)

Is there a reason not to prefill with both minimal and recommanded permissions?

**Comment by rijkvanzanten** [comment]
> Is there a reason not to prefill with both minimal and recommanded permissions?

You can change the recommended permissions at any time, as they're simply auto-generated permissions rows. However, you can't change the app minimal permissions, as those are injected based on the app_access setting on the role. In order to avoid confusion on when those checkboxes would be disabled or not, it makes more sense to simply show a message below saying that certain fields will always be accessible, based on an external setting.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
